<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>450</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(108, 108, 108);
border-radius: 10px;
font-size: 23px;</string>
    </property>
    <property name="text">
     <string>Предсказать оценку</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
background: rgb(171, 171, 171);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>   Количество часов, которые вы уделяете в неделю :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
background: rgb(171, 171, 171);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>   Самая популярная оценка оценка :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>150</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;
background: rgb(171, 171, 171);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>  Сколько часов вы спите :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 16px;
background: rgb(171, 171, 171);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>  Любите ли вы предмет? (в ответ запишите &quot;да&quot; или &quot;нет&quot;) :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>350</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(141, 141, 141);
font-size: 18px;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>  Скорей всего вы учитесь:   ?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
